qa-core/src/tests/public-api/PublicAPIClient.ts

```diff
@@ -0,0 +1,58 @@
+import env from "../../environment"
+import fetch from "node-fetch"
+
+interface HeaderOptions {
```

Comment:
Should it be `Options` given it's more than just headers?

Reply:
Yeah tbh i was thinking about getting rid altogether and having it fall back on the native `node-fetch` types.

Reply:
Would be even better 👍